packages/firebase/index.d.ts

```diff
@@ -5853,6 +5853,22 @@ declare namespace firebase.firestore {
      * CACHE_SIZE_UNLIMITED to disable garbage collection.
      */
     cacheSizeBytes?: number;
+
+    /**
+     * Forces the SDK’s underlying network transport (WebChannel) to use
+     * long-polling. Each response from the backend will be closed immediately
+     * after the backend sends data (by default responses are kept open in
+     * case the backend has more data to send). This avoids incompatibility
+     * issues with certain proxies, antivirus software, etc. that incorrectly
+     * buffer traffic indefinitely. Use of this option will cause some
+     * performance degradation though.
+     *
+     * This setting may be removed in a future release. If you find yourself
+     * using it to work around a specific network reliability issue, please
+     * tell us about it in
+     * https://github.com/firebase/firebase-js-sdk/issues/1674.
+     */
```

Comment:
Can you put a `@webonly` tag at the end of this comment? PR https://github.com/firebase/firebase-js-sdk/pull/1679 will add ability for the doc generation process to filter out symbols with this tag when generating the Node version of the reference docs.